akka-stream/src/main/scala/akka/stream/impl/Throttle.scala

```diff
@@ -29,7 +28,7 @@ private[stream] class Throttle[T](cost: Int,
     var lastTokens: Long = 0
     var previousTime: Long = 0
```

Comment:
This should start out at `scaledMaximumBurst` and `now()`—it currently depends on the fact that `now()` is sufficiently greater than zero when the first element arrives.